libmachine/host.go

```diff
@@ -256,6 +256,15 @@ func (h *Host) Upgrade() error {
 }
 
 func (h *Host) Remove(force bool) error {
+	machineState, err := h.Driver.GetState()
+	if err != nil {
+		return err
+	}
+
+	if machineState != state.Stopped {
```

Comment:
I agree with skipping the check on force, but making machines automatically remove-able on `Error` state seems risky to me.